src/swagger-specs/destination-authoring.yaml

```diff
@@ -0,0 +1,3286 @@
+
+#      security:
+#       - dest_auth:
+#            - "write:dest"
+#            - "read:dest"
+    post:
+      tags:
+        - "Destination publishing"
+      summary: "Submit a destination configuration for publishing"
+      description: "Submit a destination configuration for publishing by making a POST request to the `/destinations/publish` endpoint."
+      operationId: "publishDestination"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-sandbox-name'
+        - in: "body"
+          name: "body"
+          description: "Destination Publish request object"
+          required: true
+          schema:
+            $ref: "#/definitions/publishedDestinationRequest"
+      responses:
+        "201":
+          description: "A successful response returns HTTP status 201 with details of your newly created destination publish request."
+          schema:
+            $ref: "#/definitions/publishedDestination"
+        "400":
+          description: "Bad request. The request payload was incorrectly formatted. Check your payload formatting before trying again."
+        '401':
+          description: "Unauthorized. A mandatory header might be missing from the request or you may be missing permissions to access the resource. Verify your request before trying again."              
+        '403':
+          description: "Forbidden. The requester is not authorized to access the resource or a mandatory header might be missing from the request. Verify your request before trying again."       
+#       "405":
+#         description: "Invalid input"
+#      security:
+#       - dest_auth:
+#            - "write:dest"
+#            - "read:dest"
+  /destinations/publish/{DESTINATION_ID}:
+    get:
+      tags:
+        - "Destination publishing"
+      summary: "Get the status of a specific destination publish request by ID"
+      description: "You can retrieve detailed information about a specific destination publish request by making a GET request to the `/destinations/publish` endpoint and providing the ID of the destination for which you want to retrieve the publishing status."
+      operationId: "getPublishedDestinations"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-sandbox-name'
+        - name: "DESTINATION_ID"
+          in: "path"
+          description: "Instance ID of the destination"
+          required: true
+          type: "string"
```

Comment:
`INSTANCE_ID` was used as the path param for instance IDs previously, whereas `DESTINATION_ID` seemed to be used for destination configuration IDs. However, `INSTANCE_ID` is _also_ being used to refer to destination configuration IDs.

Is `INSTANCE_ID` just being used as a template for any resource "instance" in the API? If so, we need to update these param names to specifically describe the resource whose instance ID we are using. In this case, `CONFIGURATION_ID` might be more appropriate.